docs/wiki/deployment/configuration.md

```diff
@@ -395,6 +395,22 @@ Example:
 
 There is a strict relationship between the top-level `file_paths` key, and `yara`'s equivalent subkey.
 
+### Prometheus
+
+The `prometheus_targets` key can be used to configure Prometheus targets to be queried. The metric timestamp is taken when the target response is received.  The `prometheus_targets` parent key consists of one child key `urls` of which value contains a list target urls to be scraped.
```

Comment:
Also for the second sentance, maybe:

The `prometheus_targets` parent key consists of one child key, `urls`, which contains a list of target urls to be scraped.